components/automate-chef-io/content/docs/nodes.md

```diff
@@ -57,9 +57,72 @@ The `/nodes` endpoint supports filtering by:
 - region
 - source_id (a reference to the primary provider's node)
 - state
-- statechange_timestamp
+- statechange_timerange (supports two timestamps of type "2019-03-05T00:00:00Z")
 - status
 - tags
+- last_run_timerange (last time reported on ingested ccr: supports two timestamps of type "2019-03-05T00:00:00Z")
+- last_scan_timerange (last time reported on ingested scan: supports two timestamps of type "2019-03-05T00:00:00Z")
+- last_run_status (status on last ingested ccr)
+- last_scan_status (status on last ingested scan)
+- last_run_penultimate_status
+- last_scan_penultimate_status
```

Comment:
Can we add a comment here for the `penultimate` fields? I had to look up what 'penultimate' was to figure out what these provide. 
`last_run_penultimate_status (status on second to last ingested ccr)`
`last_scan_penultimate_status (status on second to last ingested scan)`